README.md

```diff
@@ -414,6 +414,18 @@ The current value of the setting is global in scope (i.e. applies to all
 sessions, whenever created) and is always consulted for the creation of
 any data object handle to govern that handle's cleanup behavior.
 
+Also, alternatively, the client may opt in to a special "redirect" behavior
+in which data objects' open( ) method forge a connection directly to whichever
+iRODS server is found to host the selected replica.  Data reads and
+writes will therefore happen on that alternate network route, instead of
+through the directly-connected server.  Though not the client's default
+behavior, this approach can turn out to be more efficient, particularly
+if several concurrent data uploads ("puts") and downloads ("gets") are 
+happening which might increase traffic on the client's main communication
+route with the server.  (See, in the next section, the
+client configuration setting `data_objects.allow_redirect`, which may be
```

Comment:
Prefer links to the section instead of phrases like `in the next section`.

Providing a clickable link protects against sections moving around.

For example:
```
See the client configuration setting `data_objects.allow_redirect` in [Python iRODS Client Settings File](#python-irods-client-settings-file), which may be set to `True` for the aforementioned opt-in.
```